src/main/scala/com/wavesplatform/matcher/api/MatcherApiRoute.scala

```diff
@@ -58,6 +59,14 @@ case class MatcherApiRoute(wallet: Wallet,
         .getOrElse[JsValue](JsString("")))
   }
 
+  @Path("/settings")
+  @ApiOperation(value = "Matcher Settings", notes = "Get matcher settings", httpMethod = "GET")
+  def getSettings: Route = (path("settings") & get) {
+    val priceAssets = matcherSettings.priceAssets
+    println(priceAssets)
+    complete(StatusCodes.OK -> Json.obj("settings" -> Json.obj("price-assets" -> priceAssets)))
```

Comment:
Please remove `println`, return an array of strings instead of an object and change path to `/priceAssets`